kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDownloadPanel.js

```diff
@@ -29,9 +29,6 @@ define([
             downloadSpecCache: null, // {'lastUpdateTime': <millisec.>, 'types': {<type>: <spec>}}
         },
         loadingImage: Config.get('loading_gif'),
-        shockURL: Config.url('shock'),
-        exportURL: Config.url('data_import_export'),
-        nmsURL: Config.url('narrative_method_store'),
```

Comment:
shock and export URLs are no longer used.
NMS URL is only used once to make a client, didn't seem worth keeping in the widget's memory.

(eeURL gets reused several times)